index.bs

```diff
@@ -436,6 +436,12 @@ Note: The group is <a href="https://github.com/webmachinelearning/webnn/issues/8
 
 Unlike APIs like WebGL, and WebGPU; this API does not intrinsically support custom shader authoring; and as a result is not prone to timing attacks that rely on shader caches, or other persistent data. The API builds upon pre-existing shaders and lower level primitives of the browser or the underlying OS. Web developers who interface with {{GPUDevice}} are expected to be aware of <a href="https://gpuweb.github.io/gpuweb/#privacy-user-agent-state">WebGPU compilation cache considerations</a>.
 
+The WebGPU API identifies <a href="https://gpuweb.github.io/gpuweb/#privacy-machine-artifacts">machine-specific artifacts</a> as a privacy consideration. Given the WebNN API defines means to record an ML worload onto a WebGPU-compatible {{GPUCommandBuffer}}, compute unit scheduling may under certain circumstances introduce a fingerprint. However, similarly to WebGPU, such fingerprints are identical across most or all of the devices of each vendor, mitigating the concern. Furthermore, software implementations can be used to further eliminate such artifacts.
+
+The WebNN API defines two developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation better select the most appropriate underlying execution device for the workload. [=Device type=] preference indicates the kind of device and is either "cpu" or "gpu". If this preference cannot be satisfied, an {{OperationError}} exception is thrown, thus this preference can in some cases add two bits of entropy to the fingerprint. [=Power preference=] indicates preference as related to the power consumption and is considered a hint only and as such does not increase entropy of the fingerprint.
```

Comment:
a nit: [=Device type=] is not a preference but a normative type in current spec.

Reply:
Thanks @huningxin, proposed fix in https://github.com/webmachinelearning/webnn/pull/271/commits/5185b9fbc524a8a1409d1a691676dedfd8252644

Reply:
looks good, thanks @anssiko !